config.go

```diff
@@ -18,11 +18,6 @@ type Config struct {
 	// a leader before we attempt an election.
 	ElectionTimeout time.Duration
 
-	// CommitTimeout controls the time without an Apply() operation
-	// before we heartbeat to ensure a timely commit. Due to random
-	// staggering, may be delayed as much as 2x this value.
-	CommitTimeout time.Duration
```

Comment:
Note to self - I think this is still useful as a tune-able. Need to look at how the new implementation works.